src/test/java/org/opensearch/knn/integ/ExpandNestedDocsIT.java

```diff
@@ -73,7 +74,12 @@ public class ExpandNestedDocsIT extends KNNRestTestCase {
     @After
     @SneakyThrows
     public final void cleanUp() {
-        deleteKNNIndex(INDEX_NAME);
+        try {
+            deleteKNNIndex(INDEX_NAME);
+        } catch (ResponseException e) {
+            // Index not found exception is acceptable because some test cases do not create an index
+            assert e.getMessage().contains("index_not_found_exception");
+        }
```

Comment:
Not sure why we are adding this. There is already a function: `wipeAllODFEIndices` in base ODFERestTestCase that is cleaning up all the indices. So why we need this?

Reply:
Ref: https://github.com/opensearch-project/k-NN/blob/f3d7d49a3dfe5c16a6d71dee04224fc095430255/src/testFixtures/java/org/opensearch/knn/ODFERestTestCase.java#L136-L138

and If I am not wrong in general we don't delete indices at the end. It gets deleted after every test, by the base functions.

Reply:
You are right. I just saw previous test case where we manually delete index at the end and forgot this part. Will remote it.